### PR TITLE
Adding typography content

### PR DESCRIPTION
**Pull request by colinpmacarthur**

### CODE REVIEW

File: _visual/typography.md [hunk 1]
```diff
@@ -212,24 +212,27 @@ Include intro of the two families, how we have diff packages, a bit on considera
...
+              <li>3. Source Sans Pro, Regular 400</li>
+              <li>4. Source Sans Pro, Bold 700</li>
+              <li>5. Source Sans Pro, Italic 400</li>
             </ul>
           </div>
           <aside class="usa-width-one-third usa-end-row">
             <h6 class="usa-heading-alt">Page Performance</h6>
+            <p><span class="usa-label-big">Medium</span></p>
+            <p>Exceeds ideal number of fonts by one. May negatively impact page load performance.</p>
             <img src="{{ site.baseurl }}/assets-styleguide/img/page-performance.jpg">
```
Comment (maya) [inline _visual/typography.md]: @colinpmacarthur shouldn't this be removed?

Comment (colinpmacarthur) [inline _visual/typography.md]: Fixed in 8203f00

File: _visual/typography.md [hunk 2]
```diff
@@ -390,24 +392,28 @@ Include intro of the two families, how we have diff packages, a bit on considera
...
+              <li>4. Merriweather, Light 300</li>
+              <li>5. Source Sans Pro, Regular 400</li>
+              <li>6. Source Sans Pro, Bold 700</li>
             </ul>
           </div>
           <aside class="usa-width-one-third usa-end-row">
             <h6 class="usa-heading-alt">Page Performance</h6>
+            <p><span class="usa-label-big">Medium</span></p>
+            <p>Exceeds ideal number of fonts by two. May negatively impact page load performance.</p>
             <img src="{{ site.baseurl }}/assets-styleguide/img/page-performance.jpg">
```
Comment (maya) [inline _visual/typography.md]: @colinpmacarthur should this be removed?

File: _visual/typography.md [hunk 3]
```diff
@@ -49,32 +46,35 @@ Include intro of the two families, how we have diff packages, a bit on considera
 <p class="usa-text-small">Info about what files to expect when you download.</p>
 
 <h3 class="usa-heading" id="pairings">Pairings + Styles</h3>
-<p>Guidance/overview on how our font packages were designed, etc. Something about load performance consideration and how we recommend one of the following font pages for use. A bit on the rationale for the pairings if it’s not covered an above section.</p>
+<p>To support both more contemporary and more traditional web design aesthetics, this font system offers recommended font pairings. Each pairing includes web hierarchy guidance on font family, weight, size, and spacing which express either more modern or more classical type design.</p>
+<p>Note: Some pairings require more font weights than others. While this allows more typographic expression, the use of four or more font weights will have a negative impact on page load performance. Find the balance that works for your product.</p>
```
Comment (maya) [inline _visual/typography.md]: This should be "more that four" not "four or more". Four is still ok.

Comment (colinpmacarthur) [inline _visual/typography.md]: This is @mollieru's copy, so Iet's double check with her.

Comment (colinpmacarthur) [inline _visual/typography.md]: Makes sense to me, tho.

File: _visual/typography.md [hunk 4]
```diff
@@ -13,8 +12,7 @@ Include intro of the two families, how we have diff packages, a bit on considera
 
 <div class="usa-grid-full">
   <div class="usa-width-one-half">
-    <p>Explanation of font selection Explanation of font selection Explanation of font selection. Explanation of font selection Explanation of font selection Explanation of font selection.</p>
-    <p>Explanation of font selection Explanation of font selection. Explanation.</p>
+    <p>Source Sans Pro is an open-source sans serif typeface created for legibility in UI design.  With a variety of weights that read easily at all sizes, Source Sans Pro provides clear headers as well as highly readable body text.  Its slender but open letters offer a clean and friendly simplicity. Advanced hinting allows Source Sans Pro to render well on Windows systems which run Cleartype, and across browsers and devices. Moreover, it supports a variety of languages and alphabets, including Western and European language, Vietnamese, pinyin Romanization of Chinese, and Navajo.</p>
```
Comment (maya) [inline _visual/typography.md]: I would love to include something about this font being inspired by twentieth-century American gothics, which became a staple of almost every type foundry coming after. It's a very American thing we can highlight.

"Source Sans Pro draws inspiration from the clarity and legibility of twentieth-century American gothic typeface designs. Distilling the best archetypical qualities of these models, Paul followed a rational design approach by simplifying glyph shapes by paring them to their essential form. "

http://www.fontsquirrel.com/fonts/source-sans-pro

cc @mollieru

Comment (maya) [inline _visual/typography.md]: Looks like there's an extra space at the end of sentences. can you remove?

Comment (colinpmacarthur) [inline _visual/typography.md]: Fixed the spaces in ffbb395.

File: _visual/typography.md (hunk 3 above)
Comment (mollieru) [inline _visual/typography.md]: Thanks for the correction! That sounds good. 

Copy edit: 
"more *than* four"

Comment (colinpmacarthur) [inline _visual/typography.md]: Changed in 15fa803

**Comment by mollieru** [comment]
@maya @colinpmacarthur updated per your helpful idea Maya!

**Pull request merged**